package.json

```diff
@@ -4,22 +4,22 @@
   "private": true,
   "dependencies": {
     "@shopify/polaris": "^4.19.0",
+    "@shopify/polaris-icons": "^4.5.0",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "react-scripts": "3.4.1",
+    "react-scripts": "4.0.3",
     "typescript": "^4.2.4"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
+    "eject": "react-scripts eject",
+    "lint": "eslint --max-warnings 0 --ext .tsx,.ts ./src",
```

Comment:
This is useful for devs, but can also be used in husky hooks and CI servers - I generally run a lint script, then a test script, then the build script before pushing code.